packages/ffe-form/less/checkbox.less

```diff
@@ -38,7 +40,7 @@
     }
 
     &::before {
-        border: solid 2px var(--ffe-v-checkbox-color);
+        border: solid 2px var(--square-color);
```

Comment:
Fint å bruke samme navngivning som andre variabler så --ffe-v-checkbox-square-color

Reply:
En liten forskjell her er ju att --square-color bare lever i komponenten og bruker feks `--ffe-v-checkbox-color`. avhenge av state checked/inavlid osv.

Reply:
Skjønner ikke helt hva du mener? Alle `--ffe-v` varabler er variabler som kun brukes internt i komponentene? Det er `--ffe-g` som er variabel navnene som er globalt for hele ffe

Reply:
Aaah skjønner! nå